readme.md

```diff
@@ -116,7 +125,7 @@ How is rebase different than a merge?
 
 Your Answer:
 ```text
-
+Both git rebase and git merge solve the same problem-- they integrate changes from one branch to another. However, rebasing rewrites the history by creating new commits for each existing commit in the original branch. The result is a much cleaner workflow.
```

Comment:
:+1: Great, would note that the rebase is creating new commits on top of the head of another branch creating a linear commit history